src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -618,6 +632,41 @@ def _get_schema_cls(cls):
         return FormRecognizerConnectionSchema
 
 
+class CustomStrongTypeConnection(_Connection):
+    def __init__(self, **kwargs):
+        configs = {}
+        secrets = {}
+        for k, v in self.__class__.__annotations__.items():
+            field_value = kwargs.get(k, None)
+            if v == Secret:
+                secrets[k] = field_value
+            else:
+                configs[k] = field_value
+        # For custom connection secrets are not checked? So does CustomStrongTypeConnection?
```

Comment:
Mark as TODO or simply remove. Or community may feel confusing about this comment.

Reply:
removed